apis/status/v1beta1/util.go

```diff
@@ -21,10 +21,9 @@ func dashExtractor(val string) []string {
 				b.WriteRune(chr)
 				prevDash = false
 				continue
-			} else {
```

Comment:
@maxsmythe - Would appreciate an extra set of eyes on this change.

Reply:
Looks right to me.